daycare/models.py

```diff
@@ -39,3 +47,24 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
Why the image needs a name?

Reply:
It gives the image his related name in order to handle it more easily in the case of multiply images per daycare.

Reply:
Won't it be easier to handle it with the id?

Reply:
Consider to satisfy one image per Day-Care to make things simple ,
and add image_url field to Day-Care class instead of create Image class .

Reply:
I think all the methods you suggested will be good for us, i will remove the name field.
about one image only - I think that it's not so  complicated to manage, so any image created must rely on a foreign key of daycare